compile error on gcc10

```
[ 81%] Built target fs_lib
Scanning dependencies of target net_lib
make[2]: Warning: File 'src/overlaybd/net/CMakeFiles/net_lib.dir/depend.make' has modification time 2.8 s in the future
[ 82%] Building CXX object src/overlaybd/net/CMakeFiles/net_lib.dir/curl.cpp.o
/root/wujiaxu/workspace/cpp/overlaybd/src/overlaybd/net/curl.cpp:215:6: error: ‘string’ in namespace ‘std’ does not name a type
  215 | std::string url_escape(const char *str) {
      |      ^~~~~~
/root/wujiaxu/workspace/cpp/overlaybd/src/overlaybd/net/curl.cpp:24:1: note: ‘std::string’ is defined in header ‘<string>’; did you forget to ‘#include <string>’?
   23 | #include "../timeout.h"
  +++ |+#include <string>
   24 | #include "../utility.h"
make[2]: *** [src/overlaybd/net/CMakeFiles/net_lib.dir/build.make:63: src/overlaybd/net/CMakeFiles/net_lib.dir/curl.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:729: src/overlaybd/net/CMakeFiles/net_lib.dir/all] Error 2
make: *** [Makefile:152: all] Error 2
```

--- a/src/overlaybd/net/curl.cpp
+++ b/src/overlaybd/net/curl.cpp
@@ -14,6 +14,7 @@
    limitations under the License.
 */
 #include <curl/curl.h>
+#include <string>
 
 #include "../alog.h"
 #include "../event-loop.h"

